trin-execution/metrics/grafana/datasource.yml

```diff
@@ -3,7 +3,7 @@ apiVersion: 1
 datasources:
 - name: Prometheus
   type: prometheus
-  url: http://prometheus:9090 
+  url: http://localhost:9090
```

Comment:
this doesn't look right

Reply:
Maybe I need to add another separate datasource to support both macos and linux, but what I do know, is http://prometheus:9090 doesn't work on linux

Reply:
I think because of my updates to docker-compose this should work. I don't run this on macos anymore, so if someone finds a bug we can fix it then